pymemcache/test/test_client_hash.py

```diff
@@ -186,6 +186,8 @@ def test_no_servers_left_with_commands(self):
         )
 
         result = client.get('foo')
+        assert result is None
+        result = client.set('foo', 'bar')
         assert result is False
```

Comment:
I assume you mean `client.get_many(['foo']) == {'foo': None}`? Currently it returns `{'foo': False}` (just tested). The former would make it more in line with how `.get(key)` works.

```python
result = client.get_many(['foo'])
assert result == {'foo': None}
```